tests/tt_metal/tt_fabric/fabric_router/test_routing_tables.cpp

```diff
@@ -85,7 +85,7 @@ TEST_F(ControlPlaneFixture, TestTGFabricRoutes) {
     EXPECT_GT(valid_chans.size(), 0);
     for (auto chan : valid_chans) {
         auto path = control_plane->get_fabric_route(FabricNodeId(MeshId{0}, 0), FabricNodeId(MeshId{4}, 31), chan);
-        EXPECT_EQ(path.size() > 0, true);
+        EXPECT_EQ(!path.empty(), true);
```

Comment:
Nit: For tests, I actually prefer
```suggestion
        EXPECT_GT(path.size(), 0);
```
because the error message provides more information to potentially infer what the bug is without having to run the debugger.
If you feel strongly against this then the following is also more idiomatic
```suggestion
        EXPECT_FALSE(path.empty());
```